sdk/attestation/attestation/package.json

```diff
@@ -56,61 +130,30 @@
+    "typedoc": "0.15.2",
     "typescript": "~4.2.0",
-    "typedoc": "0.15.2"
+    "util": "^0.12.1"
   },
-  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/attestation/attestation/README.md",
-  "repository": "github:Azure/azure-sdk-for-js",
-  "bugs": {
-    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
-  },
-  "files": [
-    "dist/",
-    "dist-esm/src/",
-    "types/attestation.d.ts",
-    "README.md",
-    "LICENSE"
-  ],
-  "scripts": {
-    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local",
-    "build:samples": "echo skipped",
-    "build:test": "tsc -p . && rollup -c 2>&1",
-    "check-format": "echo skip format check as this package has only auto generated code",
-    "clean": "rimraf dist dist-esm dist-test temp types *.tgz *.log",
-    "execute:samples": "echo skipped",
-    "extract-api": "api-extractor run --local",
-    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace \"dist-esm/test/{,!(browser)/**/}*.spec.js\"",
-    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint package.json api-extractor.json test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint package.json api-extractor.json src test --ext .ts",
-    "pack": "npm pack 2>&1",
-    "prebuild": "npm run clean",
-    "test": "npm run clean && npm run build:test && npm run unit-test",
-    "unit-test:browser": "karma start --single-run",
-    "unit-test:node": "mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/{,!(browser)/**/}*.spec.ts\"",
-    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
-    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --stripInternal --mode file --out ./dist/docs ./src"
-  },
-  "sideEffects": false,
-  "browser": {
-    "./dist-esm/src/utils/base64.js": "./dist-esm/src/utils/base64.browser.js",
-    "./dist-esm/src/utils/textEncoding.js": "./dist-esm/src/utils/textEncoding.browser.js",
-    "./dist-esm/test/utils/base64url.js": "./dist-esm/test/utils/base64url.browser.js",
-    "./dist-esm/test/utils/Buffer.js": "./dist-esm/test/utils/Buffer.browser.js",
-    "./dist-esm/test/utils/env.js": "./dist-esm/test/utils/env.browser.js"
+  "//sampleConfiguration": {
+    "skipFolder": false,
+    "disableDocsMs": true,
+    "productName": "Azure Attestation",
+    "productSlugs": [],
```

Comment:
any idea what values can be added here?

Reply:
None whatsoever :). THis is 100% cut&pasted from the template.

Reply:
@witemple-msft do you know what should be the strings for attestation?

Reply:
@LarryOsterman @deyaaeldeen it should be `"productSlugs": ["azure-attestation"]`

Reply:
Thank you Will, fixing in my next pull request.